aea/cli/publish.py

```diff
@@ -58,29 +65,35 @@
 
 
 @click.command(name="publish")
-@registry_flag(
-    help_local="For publishing agent to local folder.",
-    help_remote="For publishing agent to remote registry.",
-)
```

Comment:
Why is this being removed?

Reply:
I'm updating the registry selection logic, Current one seems a bit rigid and over engineered